modules/table/pom.xml

```diff
@@ -74,16 +74,23 @@
 
         <dependency>
             <groupId>org.apache.ignite</groupId>
-            <artifactId>ignite-schema</artifactId>
+            <artifactId>ignite-raft</artifactId>
             <version>${project.version}</version>
         </dependency>
 
+
+        <!-- Test dependencies -->
         <dependency>
             <groupId>org.apache.ignite</groupId>
-            <artifactId>ignite-raft</artifactId>
+            <artifactId>ignite-raft-client</artifactId>
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>metastorage-client</artifactId>
```

Comment:
Why we need explicit metastorage-client dependency? I believe we should use metastorageManager instead.